packages/eyes/src/index.ts

```diff
@@ -72,21 +109,34 @@ async function waitForTestsCompletion(page: puppeteer.Page, url: string):
 
   // TODO: add a timeout for each test to make sure we terminate if they get
   // stuck.
+  let testsPassed: boolean = true;
+  consoleLog('Format: <component simName>');
   for (const [i, {title}] of tests.entries()) {
+    let result;
     await page.evaluate(`puppeteerRenderTest(${i})`);
     const screenshot = await page.screenshot();
-    // TODO: Compare the screenshot using eyes
-    console.log({title, screenshotBufferSize: screenshot.byteLength});
+    await eyes.open(projectName, title, null);
+    const {asExpected} = await eyes.checkImage(screenshot);
```

Comment:
What does checkImage return in the case of a new image?

Reply:
`asExpected` is false in this case, so I added a check to ensure that the component isn't new